easybuild/easyconfigs/v/vaspkit/vaspkit-1.3.3.eb

```diff
@@ -0,0 +1,33 @@
+# IT4INNOVATIONS 2022
+# JK
+
+easyblock = 'Tarball'
+
+name = 'vaspkit'
+version = '1.3.3'
+
+homepage = 'https://vaspkit.com/'
+description = """VASPKIT aims at providing a powerful and user-friendly interface to perform high
+ throughput analysis of various material properties from the raw calculated data using the widely-used
+ VASP code."""
+
+toolchain = SYSTEM
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(namelower)s.%(version)s.linux.x64.tar.gz']
+patches = ['vaspkit-1.3.3-fix-setup-script.patch']
+checksums = [
+    'e428d5e3f3dde5f11fc17770f7c0864a',  # vaspkit.1.3.3.linux.x64.tar.gz
+    'f4f523c04d27c9f2165b140a0c0ccde0',  # vaspkit-1.3.3-fix-setup-script.patch
+]
+extract_sources = True
+
+dependencies = [
+    ('matplotlib', '3.3.3', '', ('foss', '2020b')),
```

Comment:
@anselmicz Although there's no actual compilation going on here, only unpacking a tarball (and applying the patch), it's better to use `foss` as `toolchain` above, rather than having a "hidden" dependency on `foss` through `matplotlib`.

So, I suggest to make 3 changes:

* rename the file to `vaspkit-1.3.3-foss-2020b.eb`
* use `toolchain = {'name': 'foss', 'version': '2020b'}` above
* change this line to `('matplotlib', '3.3.3')`

Reply:
> @anselmicz Although there's no actual compilation going on here, only unpacking a tarball (and applying the patch), it's better to use `foss` as `toolchain` above, rather than having a "hidden" dependency on `foss` through `matplotlib`.
> 
> So, I suggest to make 3 changes:
> 
>     * rename the file to `vaspkit-1.3.3-foss-2020b.eb`
> 
>     * use `toolchain = {'name': 'foss', 'version': '2020b'}` above
> 
>     * change this line to `('matplotlib', '3.3.3')`

sure, no problem